<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WeighUI</class>
 <widget class="QMainWindow" name="WeighUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>采样前</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="text">
            <string>第一次</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_2">
           <property name="text">
            <string>第二次</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>采样后</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QRadioButton" name="radioButton_3">
           <property name="text">
            <string>第三次</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton_4">
           <property name="text">
            <string>第四次</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">radioGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="printPartBtn">
      <property name="text">
       <string>打印选定区域</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="openBtn">
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="printBtn">
      <property name="text">
       <string>全部打印</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="4">
     <widget class="MyTableView" name="tableView">
      <property name="minimumSize">
       <size>
        <width>9</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="4">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>样品类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sampleTypeLine">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>滤膜类型：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mediumTypeBox">
        <property name="currentText">
         <string/>
        </property>
        <property name="currentIndex">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>颗粒物</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>水质SS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>海水SS</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>环境温度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="tempratureLine">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>环境湿度：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="humidityLine">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="saveBtn">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>636</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyTableView</class>
   <extends>QTableView</extends>
   <header location="global">mytableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="radioGroup"/>
 </buttongroups>
</ui>
